doskvol-tools/src/types/TableVariant.ts

```diff
@@ -0,0 +1,6 @@
+export interface TableVariant {
+    tableTitle: Variant,
+    rarity: number
+}
+
+export type Variant = `common` | 'rare'
```

Comment:
This have different string syntax. Should it be just `'` or `"`

https://www.typescriptlang.org/docs/handbook/literal-types.html